apps/mull-ui/src/app/app.tsx

```diff
@@ -41,6 +44,15 @@ export const App = () => {
             <EventPage event={dummyEvent} prevPage={'Review'} />
           </div>
         </Route>
+        {/* Without this outter Route component, the inner components would show on all routes that are not matched */}
+        <Route exact path={[`${ROUTES.DISCOVER}`, `${ROUTES.UPCOMING}`, `${ROUTES.MYEVENTS}`]}>
```

Comment:
Is there a reason why we can't have:
```suggestion
        <Route exact path={[ROUTES.DISCOVER, ROUTES.UPCOMING, ROUTES.MYEVENTS]}>
```
I tried it and it seems to work